theano/tensor/elemwise.py

```diff
@@ -1323,13 +1323,13 @@ class CAReduce(Op):
     CAReduce(mul) -> product
     CAReduce(maximum) -> max
     CAReduce(minimum) -> min
-    CAReduce(or) -> any # not lazy
-    CAReduce(and) -> all # not lazy
+    CAReduce(or_) -> any # not lazy
```

Comment:
You should make this a list, as you would have noticed if you generated the html.